### ISSUE TITLE
[FEATURE REQUEST] Adjust output image to include Append skills between Active Skills and CC'd Command Cards

### ISSUE THREAD

**Issue by TohnoCoding** [rayshift/rayshift#14]
With the early implementation of Servant Coins in NA, mechanics have essentially caught up between both JP and NA, so I consider it perfectly viable to account for the existence of Append Skills in the Rayshift output images.

For my own personal usage I came up with this solution in Photoshop:
![image](https://user-images.githubusercontent.com/98242280/174842933-8e09eced-c22f-4b3a-bddd-5d00106053be.png)

But I believe it would be more aesthetically pleasing and appropriate to locate the row with the Appends right above the Active Skills, and the command cards with any command codes above that. Yes, it'd take a little more space of the Servant artwork, but I believe it's a fair trade for more easily accessible information.

In that vein, I believe the same logic could apply to the live display of the queried player's decks on the site, though not knowing the backend and how the dom entities are constructed, I can't estimate the amount of work required to implement this.

Food for thought.

**Comment by EquinoxPhoenix** [comment]
Yes this, I have been wondering since I started using the site why I can't get both Active and Append Skills showing together.

**Comment by TohnoCoding** [comment]
Has there been any news on the development of this feature? Just curious; it's been over two years since I initially opened the issue and over four months since someone self-assigned this. This is not intended as a way to pressure, I'm genuinely wondering if there's been any movement.

This was achieved with some really crude editing of the existing output, but something akin to this, perhaps?
![image](https://github.com/user-attachments/assets/d0f43ba0-e39d-43ac-9de3-34dcbc2fc3d7)

**Comment by neobenedict** [comment]
How about this?

![image](https://github.com/user-attachments/assets/40ef66ed-fda7-41c3-865c-04fb1cc77d88)


```

/* site.min.css | https://rayshift.io/css/site.min.css?v=5Jo9eCtpIxtq5st1jDHYR6gceR7SaT7C1IJwn9ZoY5E */

.bc {
  /* font-size: .9vw; */
  /* line-height: 95%; */
  font-size: .9vw !important;
  line-height: 75%;
  flex-direction: column;
}

.skill-levels-main {
  /* flex: 55; */
}

.skill-levels-alt {
  /* flex: 55; */
}

.np-level {
  /* position: fl; */
  /* position: sticky; */
  position: absolute;
  right: 3px;
  margin-top: -25px;
  height: 100%;
}

.np-icon {
  /* width: 100%; */
  /* height: 100%; */
  width: 18px;
  height: 20px;
}

.bc-bottom {
  flex-direction: row !important;
  padding-top: 0 !important;
}

/* Element | https://rayshift.io/ */

#deck-1 > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(3) > div:nth-child(2) {
  /* font-size: 26.4px; */
}

/* Element | https://rayshift.io/ */

#deck-1 > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(2) > div:nth-child(2) {
  padding: 3px 0 3px 0;
}
```

I will add it soon, completely forgot.

**Comment by TohnoCoding** [comment]
That works wonderfully for all the required purposes!

**Comment by EquinoxPhoenix** [comment]
Oh yes please this would be a literal godsend

**Comment by TohnoCoding** [comment]
For what it's worth, I did some live editing on the page and it came out surprisingly decent. (I'm only specifying what I changed in each of the classes.)

```CSS
.skill-levels-alt {
    /* display: none; */
}

.skill-levels {
    /* flex: 75; */
}

.bc {
    /* flex: 0 0 auto; */
    flex-direction: column;
}

.bc-bottom {
    font-size: .9vw !important;
}

.np-level {
    /* flex: 25; */
    position: absolute;
    right: 1.1em;
    margin-top: 0.8vh;
}

.servant-hp-atk {
    font-size: .9vw !important;
}

.np-icon {
    width: 18px;
    height: 20px;
}
```

Not sure how that may be applied to the image generator but the edits I made make the live page look very consistent.

![image](https://github.com/user-attachments/assets/e3c90cea-aa71-4ffa-84c1-bc8605de4627)

_**EDIT:**_ Should mention that I removed all in-line applications of `style="display: none;"` from all elements for this to work, because for whatever reason, even though `skill-levels-alt` already applies `display: none;`, it's also being applied in-line to all the append skill level divs on top of the class.
